### DISCUSSION TITLE
Purchasable maps of the level ahead.

### DISCUSSION THREAD

**Discussion by Cogs16** [FakeFishGames/Barotrauma#5670] [Ideas and suggestions]
TLDR Outposts along a certain route could stock maps of the route ahead. This would make navigation more in-depth, as captains would also have to search for landmarks such as formations and even beacon stations. 

Other Possibilities
-Levels could be more nebulous with more dead ends, making map reading important
-Outposts might not display their location, or may require you to be within 1 kilometer in order to see them. 
-If the route has a functioning beaconstation, distances to the outposts are shown, perhaps an optimal route is even displayed somewhere, making them more functional ingame.
-Captains would have to decide whether or not its worth it to purchase a map, adding more intense decisionmaking. 
-Lastly, maps could display dangers and possible routes around them (i.e the hammer head swarm is here, but here is another route around it)
